app/containers/message/hooks/useMediaAutoDownload.tsx

```diff
@@ -150,7 +150,7 @@ export const useMediaAutoDownload = ({
 			download();
 			return;
 		}
-		if (!showAttachment || !currentFile.title_link || isEncrypted) {
+		if (!showAttachment || isEncrypted) {
```

Comment:
Everything we display is downloaded locally first and the uri is persisted on `message.attachments[x].title_link` https://github.com/RocketChat/Rocket.Chat.ReactNative/blob/ddf1f3ffac7e858f04f63c43485a5911c341d02e/app/lib/methods/handleMediaDownload.ts#L213
I don't think we should allow displaying `showAttachment` if there's no title_link here.

There could be another bug though.
There's one where the image loading indicator keeps spinning, but the file was already downloaded and tapping the indicator opens AttachmentView correctly.

Reply:
I have added few conditions to download the image before displaying it, so this condition is back :)